#! /bin/sh /usr/share/dpatch/dpatch-run
## 03_gui.dpatch by Jelmer Vernooij <jelmer@samba.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Fix dir seperator for POSIX

@DPATCH@

diff -ur nunit-2.2.8/src/GuiComponents/UiKit/nunit.uikit.build nunit-2.2.8-orig/src/GuiComponents/UiKit/nunit.uikit.build
--- nunit-2.2.8/src/GuiComponents/UiKit/nunit.uikit.build	2006-02-23 19:21:22.000000000 +0100
+++ nunit-2.2.8-orig/src/GuiComponents/UiKit/nunit.uikit.build	2006-07-13 19:15:59.000000000 +0200
@@ -20,6 +20,9 @@
       <references basedir="${current.build.dir}">
         <include name="nunit.core.dll"/>
         <include name="nunit.util.dll"/>
+		<include name="System.Windows.Forms.dll"/>
+		<include name="System.Data.dll"/>
+		<include name="System.Drawing.dll"/>
       </references>
     </csc>
   </target>
diff -ur nunit-2.2.8/src/GuiRunner/nunit-gui/nunit-gui.build nunit-2.2.8-orig/src/GuiRunner/nunit-gui/nunit-gui.build
--- nunit-2.2.8/src/GuiRunner/nunit-gui/nunit-gui.build	2005-12-09 08:43:52.000000000 +0100
+++ nunit-2.2.8-orig/src/GuiRunner/nunit-gui/nunit-gui.build	2006-07-13 19:16:18.000000000 +0200
@@ -23,6 +23,8 @@
         <include name="nunit.uikit.dll"/>
         <include name="nunit.core.dll"/>
         <include name="nunit.framework.dll"/>
+		<include name="System.Windows.Forms.dll"/>
+		<include name="System.Drawing.dll"/>
       </references>
     </csc>
   </target>
